pkgs/development/python-modules/sslib/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "sslib";
+  version = "0.2.0";
+  format = "wheel";
```

Comment:
I saw only now that PyPI only has wheels. We prefer building from source. Please ask upstream to publish a source tarball or go back to using `fetchFromGitHub`. Sorry!

Reply:
Thanks, I added the missing source archive to PyPI and updated the derivation accordingly.